Update GRC descriptions (#6243)
open-cluster-management/rhacm-docs: governance/policy_overview.adoc

| The name of the security control that is being checked.
For example, Access Control or System and Information Integrity.

| `spec.disabled`
| Required
| Set the value to `true` or `false`. The `disabled` parameter provides the ability to enable and disable your policies.

| `spec.remediationAction`
| Optional
| Specifies the remediation of your policy. The parameter values are `enforce` and `inform`. If specified, the `spec.remediationAction` value that is defined overrides any `remediationAction` parameter defined in the child policies in the `policy-templates` section. For example, if the `spec.remediationAction` value is set to `enforce`, then the `remediationAction` in the `policy-templates` section is set to `enforce` during runtime.

| `spec.copyPolicyMetadata`
| Optional

> | Specifies whether a policy's labels and annotations should be copied when replicating the policy to a managed cluster. If set to true (default), all the policy's labels and annotations will be copied to the replicated policy. If set to false, only the policy framework specific policy labels and annotations will be copied to the replicated policy.

Reviewer: ```suggestion
| Specifies whether the labels and annotations of a policy should be copied when replicating the policy to a managed cluster. If you set to `true`, all of the labels and annotations of the policy are copied to the replicated policy. If you set to `false`, only the policy framework specific policy labels and annotations are copied to the replicated policy.
```
